cmd/minikube/cmd/start_flags.go

```diff
@@ -193,6 +194,7 @@ func initDriverFlags() {
 	startCmd.Flags().String(kvmQemuURI, "qemu:///system", "The KVM QEMU connection URI. (kvm2 driver only)")
 	startCmd.Flags().Bool(kvmGPU, false, "Enable experimental NVIDIA GPU support in minikube")
 	startCmd.Flags().Bool(kvmHidden, false, "Hide the hypervisor signature from the guest in minikube (kvm2 driver only)")
+	startCmd.Flags().Int(kvmNUMACount, 1, "Simulate numa node count in minikube, supported numa node count range is 1-8 (kvm2 driver only)")
```

Comment:
Since the default policy on kubernetes is a "none policy", as described [here](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policies), I think we should set the default for this flag to 0

Reply:
by default the numa count is 1

```
❯ numactl -H
available: 1 nodes (0)
node 0 cpus: 0 1 2 3 4 5 6 7 8 9 10 11
node 0 size: 64329 MB
node 0 free: 54718 MB
node distances:
node   0 
  0:  10 
```